src/events/crawler/get-sources/load-sources.js

```diff
@@ -1,6 +1,25 @@
 import fastGlob from 'fast-glob';
 import join from '../../../shared/lib/join.js';
 import log from '../../../shared/lib/log.js';
+import * as countryLevels from '../../../shared/lib/geography/country-levels.js';
+
+/** Check location inclusion, based on command-line options.
+ * (*) location - the location
+ * (*) opts - options
+ */
+function includeLocation(location, opts) {
+  const relpath = location._path.replace(/.*?src.shared.scrapers./, '');
+
+  if (opts.skip && relpath.startsWith(opts.skip)) return false;
+
+  if (opts.location && !relpath.startsWith(opts.location)) return false;
+
+  // select location based on country level id
+  // for example "yarn start -i id3:AU-VIC"
+  if (opts.id && opts.id !== countryLevels.getIdFromLocation(location)) return false;
```

Comment:
So I'd remove this id. I added it and there is clearly no point of doing this one anymore.

Reply:
Got it, removing, and checking timeseries run-crawler as well.